test/NuGet.Clients.Tests/NuGet.VisualStudio.Implementation.Test/Extensibility/VsFrameworkCompatibilityTests.cs

```diff
@@ -52,6 +52,24 @@ public void VsFrameworkCompatibility_GetNearestRejectsNullFallbackFrameworks()
             Assert.Throws<ArgumentNullException>(() => target.GetNearest(targetFramework, fallbackTargetFrameworks, frameworks));
         }
 
+        [Fact]
+        public void VsFrameworkCompatibility_GetNearestNet5EraFails()
```

Comment:
Why do we want this to fail?  Isn't there a risk that a bunch of VS components and extensions will needlessly fail with net5.0 projects?

Reply:
This API deals directly with `FrameworkName` objects, which explicitly do not support `platform`. My previous iteration on this patch actually made this return null, but some feedback from @nkolev92 convinced me to change it so the error would be LOUD

Reply:
ok, works for me.